docs/src/pages/components/data-grid/style/StylingRowsGrid.tsx

```diff
@@ -1,14 +1,14 @@
 import * as React from 'react';
 import { DataGrid } from '@material-ui/data-grid';
 import { useDemoData } from '@material-ui/x-grid-data-generator';
-import { createMuiTheme, darken, lighten, Theme } from '@material-ui/core/styles';
+import { createTheme, darken, lighten, Theme } from '@material-ui/core/styles';
```

Comment:
If we do this change, we need to change https://github.com/mui-org/material-ui-x/blob/30512081208da6b47922950206bc72cf18254c24/packages/grid/data-grid/package.json#L48

Reply:
I just noticed the problem and since the docs use the latest version I thought it is safe to make the change.

Reply:
Ok, then I would label this as a BC: drop support of v4 < 4.12.0, and update the peer dependency scope.

Reply:
Updated